README.rst

```diff
@@ -29,16 +29,18 @@
    </p>
 
 
-GETTSIM aims at providing a depiction of the German Taxes and Transfers System that is
-usable across a wide range of research applications, ranging from highly complex dynamic
+GETTSIM provides a depiction of the German Taxes and Transfers System that will be
+usable in a wide array of research applications, ranging from highly complex dynamic
 programming models to extremely detailed microsimulation studies.
 
 GETTSIM is implemented in Python, thereby achieving both user friendliness and
 flexibility. All features are extensively tested.
 
-The current version is usable, but probably only by the initiated. Please get in touch
-(ideally by joining our Zulip Chat at https://gettsim.zulipchat.com) if you want to
-use it already, else keep an eye out for what there is to come!
+The current version is usable and should yield reasonably correct results. Development
+has valued correctness over speed so far. Hence, GETTSIM is rather slow at this point in
```

Comment:
> has valued correctness over speed so far

this sounds a bit as if we had a correctness vs. speed trade-off and we would later go for speed and throw correctness away which is of course wrong.

Reply:
... has focused on correctness until now; improving its performance is on our schedule for 2020. ?